manifests/deis-router-rc.yaml

```diff
@@ -23,3 +23,7 @@ spec:
         ports:
         - containerPort: 80
           hostPort: 80
+        - containerPort: 443
+          hostPort: 443
+        - containerPort: 2222
```

Comment:
> who is in charge of Ingress, please make it happen

This concept exists in k8s.  It's actually _called_ ingress, but it's beta, and available implementations of "ingress controllers" are alpha.  (This router is inspired by the nginx-alpha ingress controller.)

And this would be a big part of why ingress isn't stable yet-- it doesn't at all address how ingress for anything other than 80/443 is handled... yet.  It is meant to eventually.